substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSubsectionBuilder.java

```diff
@@ -75,6 +96,27 @@ private void build(ClassEntry classEntry) {
         for (PrimaryEntry primaryEntry : classEntry.getPrimaryEntries()) {
             build(primaryEntry);
         }
+        /* Add manifested static fields as S_GDATA32 records. */
+        final short sectionId = (short) ((PECoffObjectFile.PECoffSection) rwSection).getSectionID();
+        classEntry.fields().filter(CVSymbolSubsectionBuilder::isManifestedStaticField).forEach(f -> {
+            int typeIndex = cvDebugInfo.getCVTypeSection().getIndexForPointer(f.getValueType());
+            String externName = memberNameToCodeViewName(classEntry, f);
+            if (cvDebugInfo.useHeapBase()) {
+                /* REL32 offset from heap base register. */
+                addToSymbolSubsection(new CVSymbolSubrecord.CVSymbolRegRel32Record(cvDebugInfo, externName, typeIndex, f.getOffset(), cvDebugInfo.getHeapbaseRegister()));
+            } else {
+                /* Offset from heap begin. */
+                addToSymbolSubsection(new CVSymbolSubrecord.CVSymbolGData32Record(cvDebugInfo, externName, HEAP_BEGIN_NAME, typeIndex, f.getOffset(), sectionId));
```

Comment:
Same way you are using `SectionName.SVM_HEAP` above you could use `DwarfDebugInfo.HEAP_BEGIN_NAME` here.

Reply:
Done - it's more clear as you suggest.